dora/core/common/src/main/java/alluxio/Constants.java

```diff
@@ -234,11 +234,5 @@ public final class Constants {
   public static final String MEDIUM_HDD = "HDD";
   public static final String MEDIUM_SSD = "SSD";
 
-  /**
-   * Please use this switch enable/disable Dora write support in development.
-   * This will be removed when Dora write support is production ready.
-   */
-  public static final boolean ENABLE_DORA_WRITE = true;
```

Comment:
fyi static final variables cannot be mocked so it's hard to test. A better way is using a method
```
public static boolean isWriteEnabled() {return ENABLE_DORA_WRITE;}
```
A static method can be mocked (although it's not the best way either). A property is probably the ideal solution.

Reply:
This PR is to remove it.

This was introduced as a static switch when doing development.
No automation test is needed at that moment.
Manually changing the value is used in development cycle.